### PR TITLE
Add failsafe as dependency

### PR DESCRIPTION
**Pull request by sfc-gh-japatel**
Just pom change to trigger whitesource.

### CODE REVIEW

**Review by sfc-gh-mnaides — APPROVED** [review]

**Review by sfc-gh-tjones — APPROVED** [review]
Nice PR title type :)

**Comment by sfc-gh-japatel** [comment]
> Nice PR title type :)

Oops, NSFW.

File: pom.xml [hunk 1]
```diff
@@ -424,6 +424,13 @@
             <version>31.0.1-jre</version>
         </dependency>
 
+        <!-- https://github.com/failsafe-lib/failsafe-->
```
Comment (sfc-gh-tzhang) [inline pom.xml:427]: Do we need it for ingest SDK to trigger white source?

**Review by sfc-gh-tzhang — APPROVED** [review]

**Comment by sfc-gh-tzhang** [comment]
> > Nice PR title type :)
> 
> Oops, NSFW.

haha, took the screenshot

**Comment by sfc-gh-japatel** [comment]
FYI, Got approval from legal and security team. ready to push this.

File: pom.xml (hunk 1 above)
Comment (sfc-gh-japatel) [reply pom.xml:427]: This PR triggered whitesource with no vuln. so i think we are good to go..

Comment (sfc-gh-japatel) [reply pom.xml:427]: (This dependency is not for triggerring whitesource, this is for retries, fallback and circuitbreaker)
Checkout: https://github.com/failsafe-lib/failsafe

**Pull request merged**